src/main/java/org/gridsuite/shortcircuit/server/entities/FeederResultEntity.java

```diff
@@ -66,4 +71,21 @@ public FeederResultEntity(String connectableId, double current, FortescueResultE
         this.current = current;
         this.fortescueCurrent = fortescueCurrent;
     }
+
+    public boolean match(ResourceFilter filter) {
+        // FeederResultEntity may only be filtered through connectableId
+        if (filter.column().equals(FeederResultEntity.Fields.connectableId) &&
+            filter.dataType() == TEXT) {
+            switch (filter.type()) {
+                case EQUALS: return connectableId.equals(filter.value().toString());
+                case CONTAINS : return connectableId.contains(filter.value().toString());
+                case STARTS_WITH: return connectableId.startsWith(filter.value().toString());
+                case NOT_EQUAL: return !connectableId.equals(filter.value().toString());
+                default:
+                    break;
+            }
+        }
+
+        return false;
+    }
```

Comment:
I don't think this should be implemented here
Maybe put it in some existing service or create a new one ?

Reply:
True, but fortunately JPA filtering makes this function useless: removed.